cycode/cli/files_collector/commit_range_documents.py

```diff
@@ -104,19 +104,27 @@ def collect_commit_range_diff_documents(
     return commit_documents_to_scan
 
 
-def calculate_pre_receive_commit_range(branch_update_details: str) -> Optional[str]:
+def calculate_pre_receive_commit_range(repo_path: str, branch_update_details: str) -> Optional[str]:
```

Comment:
Do we need to pass repo path to the method call here: https://github.com/cycodehq/cycode-cli/blob/main/cycode/cli/apps/scan/pre_receive/pre_receive_command.py#L50 ?